cypress-custom/integration/limit-orders.test.ts

```diff
@@ -0,0 +1,27 @@
+const CHAIN_ID = 5
+
+function pickToken(symbol: string, role: 'input' | 'output') {
+  cy.get(`#limit-orders-currency-${role} .open-currency-select-button`).click()
+  cy.get('#token-search-input').type(symbol)
+  cy.get('#currency-list').get('div').contains(symbol).click({ force: true })
+}
+
+describe('Limit orders', () => {
+  it('Confirmation modal must contains values that were entered while creating', () => {
+    const inputAmount = '0.001'
+    const outputAmount = '90000000000000'
```

Comment:
I know this is goerli, but what about using more reasonable values, like 1 and 2000?

Unless... your goal is to test the formatted value on line 25?

Reply:
I set the real market values.

>your goal is to test the formatted value on line 25?

No :) It's side-effect